### ISSUE TITLE
WGL: ESP not properly saved

### ISSUE THREAD

**Issue by pnx** [Dav1dde/glad#240]
I get this error:
> Run-Time Check Failure #0 - The value of ESP was not properly saved across a function call.  This is usually a result of calling a function declared with one calling convention with a function pointer declared with a different calling convention.

Tracked the error down to the first call to `glad_wgl_get_proc_from_userptr` with `wglGetProcAddress` as userptr (standard glad wgl loader).

it needs to be called with `__stdcall` for some reason that i dont know. So you either have to modify `glad_wgl_get_proc_from_userptr`:

```c
static GLADapiproc glad_wgl_get_proc_from_userptr(void *userptr, const char* name) {
    return (GLAD_GNUC_EXTENSION (GLADapiproc (__stdcall*)(const char *name)) userptr)(name);
}
````

or define your own function that calls `wglGetProcAddress`:

```c
GLADapiproc func_loader(const char *name) {
	return (GLADapiproc) wglGetProcAddress(name);
}
```

Is this indeed a bug or can someone with abit more knowledge point out that I am doing something wrong :P

I am compiling the [WGL example](https://github.com/Dav1dde/glad/blob/glad2/example/c/wgl.c) with VS 2017 (v141)

**Comment by Dav1dde** [comment]
I assume you're on or compiling for 32bit Windows?

**Comment by pnx** [comment]
> I assume you're on or compiling for 32bit Windows?

Yes that is correct. I am on Windows 10 64-bit and compiling for x86

**Comment by Dav1dde** [comment]
Should be fixed in abf0099cf23ba5dea6749c9321a88a33c977593e, please test if possible (web generator has already been updated)

**Comment by pnx** [comment]
Works :)
